distribution/src/main/gateway/bin/gateway.start

```diff
@@ -56,5 +56,15 @@ fi
 # Add a directory for management support
 JAVA_LIBRARY_PATH="$GW_HOME/lib/sigar"
 
-exec java $GATEWAY_OPTS -Djava.library.path="$JAVA_LIBRARY_PATH" -XX:+HeapDumpOnOutOfMemoryError -cp "$GW_HOME/lib/*" org.kaazing.gateway.server.Main $*
+# Verify if the gateway identifier was provided (required by Agorna multiple gateway instances)
+if [ "$GATEWAY_IDENTIFIER" == "" ] && [[ $GATEWAY_OPTS == *"org.kaazing.gateway.management.AGRONA_ENABLED=true"* ]]; then
```

Comment:
What if $GATEWAY_IDENTIFIER is set and "org.kaazing.gateway.management.AGRONA_ENABLED=true" is not present in GATEWAY_OPTIONS? If I understand the code correctly we would in this case go down the else branch and start the metrics server, which seems wrong.
What is the exact intention of this script change?